cue.c

```diff
@@ -481,6 +481,11 @@ struct cue_sheet *cue_from_file(const char *file)
 	char *buf = mmap_file(file, &size);
 	if (size == -1)
 		return NULL;
+
+	// ignore BOM
+	if (size >= 3 && memcmp(buf, "\xEF\xBB\xBF", 3) == 0)
+		buf += 3;
```

Comment:
Isn't the `size` passed to `cue_parse()` now off by 3?

Reply:
Thanks! My c is a bit rusty so I appreciate that catch, and I can confirm it was also leaking memory when it came time to `munmap`.